runtime/near-vm-runner-standalone/example-contract/Cargo.toml

```diff
@@ -12,7 +12,8 @@ serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 wee_alloc = "0.4.5"
 
-near-sdk = "0.6.0"
+#near-sdk = "0.6.0"
+near-sdk = { path = "/Users/nike/near/near-sdk-rs/near-sdk" }
```

Comment:
Is this a local path?  Doesn't the params estimator clone this repo?

Reply:
Yeah, that fix shalln't be committed, it's a local testing artifact.